std_msgs/UInt32 nb_loops
---
bool success
string message
